src/nl/surf/eduhub/validator/service/main.clj

```diff
@@ -35,5 +37,9 @@
 (defn -main [& _]
   (let [config (config/validate-and-load-config env)]
     ;; set config as global var (read-only) so that the workers can access it
-    (reset! config/config-atom config)
+    (w/start (assoc w/default-opts
+               :broker (broker/new-consumer broker/default-opts)
+               :middleware (fn [next]
+                             (fn [opts job]
+                               (next (assoc opts :config config) job)))))
```

Comment:
wat zit er in opts hier? hoe verhoudt dat zich tot de argumenten die `validate-endpoint` krijgt?

ik zou hier denken dat je hier eigenlijk iets met `job` zou moeten doen... m.a.w. weet je zeker dat dit werkt, want je unit tests testen dit stukje niet